<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTicketDialog</class>
 <widget class="QDialog" name="AddTicketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>81</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>160</width>
     <height>240</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="Label_4">
      <property name="text">
       <string>Brand Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="brandLineEdit"/>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="Label">
      <property name="text">
       <string>Model Item</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modelLineEdit"/>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="Label_2">
      <property name="text">
       <string>Registration ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="registrationLineEdit"/>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>58</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="Label_3">
      <property name="text">
       <string>Problem Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="problemLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>158</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Assign Employee:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTicketDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTicketDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
